resource-server/README.md

```diff
@@ -0,0 +1,57 @@
+# Okta Spring Security Resource Server Example
+
+This sample application authenticates requests against your Spring application, using access tokens.
+
+The access tokens are obtained via the [Implicit Flow][].  As such, you will need to use one of our front-end samples with this project.  It is the responsibility of the front-end to authenticate the user, then use the obtained access tokens to make requests to this resource server.
+
+## Prerequisites
+
+Before running this sample, you will need the following:
+
+* An Okta Developer Account, you can sign up for one at https://developer.okta.com/signup/.
+* An Okta Application, configured for Singe-Page App (SPA) mode. This is done from the Okta Developer Console and you can find instructions [here][OIDC SPA Setup Instructions].  When following the wizard, use the default properties.  They are are designed to work with our sample applications.
+* One of our front-end sample applications to demonstrate the interaction with the resource server:
+  * [Okta React Sample Apps][]
+  * [Okta Angular Sample Apps][]
+
+A typical resource-server requires a frontend and a backend application, so you will need to start each process:
+
+**backend:**
+```bash
+cd resource-server
+mvn -Dokta.oauth2.issuer=https://{yourOktaDomain}.com/oauth2/default
+```
+
+**front-end:**
+
+Instead of using one of our front-end sample applications listed above, you can also use a front-end within this repo to quickly test the resource server.
+
+This project is just serving a static [`index.html`](front-end/src/main/resources/static/index.html) so you WILL need to update the configuration in that file, see line 53:
```

Comment:
Line 53 is [incorrect](https://github.com/okta/samples-java-spring/blob/master/front-end/src/main/resources/static/index.html#L53). I think it should be `/front-end/src/main/resources/static/js/config.js`

Reply:
Updated